docs/src/docs/integrations/k8s-init-container.md

```diff
@@ -0,0 +1,193 @@
+# Deploying a ModelKit to Kubernetes with the Kit Init Container
+
+This tutorial shows you how to fetch, verify, and unpack a ModelKit into your Kubernetes Pod using the Kit init container image `ghcr.io/kitops-ml/kitops-init:latest`.
+
+## Overview
+
+The Kit init container lets you:
+
+1. **Fetch** a ModelKit from an OCI registry.
+2. **Verify** its signature via Sigstore [Cosign][cosign] (optional).
+3. **Unpack** it to a shared volume.
+4. **Use** it in your application container just like any local files.
+
+## Prerequisites
+
+- A working Kubernetes cluster (Minikube, Kind, GKE, EKS, etc.)
+- `kubectl` [configured](https://kubernetes.io/docs/tasks/tools/) to talk to that cluster
+- A ModelKit published at an OCI registry (e.g. `ghcr.io/kitops-ml/my-modelkit:latest`)
+- *(Optional)* Cosign key or OIDC issuer for signature verification
+
+## 1. Define a Shared Volume
+
+Use an `emptyDir` volume so both containers share the unpacked files:
+
+```yaml
+volumes:
+  - name: modelkit-storage
+    emptyDir: {}
+```
```

Comment:
Maybe we need a note/warning about ephemeral storage size, as this could fail kind of weirdly for larger models.

Reply:
Would a link to https://kubernetes.io/docs/concepts/storage/volumes/#emptydir work?